src/main/java/de/retest/recheck/report/ResultWithError.java

```diff
@@ -0,0 +1,79 @@
+package de.retest.recheck.report;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+import de.retest.recheck.report.action.ActionReplayData;
+import de.retest.recheck.report.action.DifferenceRetriever;
+import de.retest.recheck.report.action.ErrorHolder;
+import de.retest.recheck.report.action.WindowRetriever;
+import de.retest.recheck.ui.actions.ExceptionWrapper;
+import de.retest.recheck.ui.actions.TargetNotFoundWrapper;
+import de.retest.recheck.ui.diff.ElementDifference;
+import de.retest.recheck.ui.diff.LeafDifference;
+
+@XmlRootElement( name = "action" )
+@XmlAccessorType( XmlAccessType.FIELD )
+public class ResultWithError extends ActionReplayResult {
```

Comment:
Upon looking at the usages, is it even possible for a ResultWithError to be constructed? As its only usage is in the `TestReportFilter`. You even made the constructor package private.

If not, can we delete this class completely? 😄 TNFE Flashbacks...